src/main/java/org/generationcp/middleware/service/impl/inventory/LotServiceImpl.java

```diff
@@ -135,47 +136,68 @@ public void updateLots(final List<ExtendedLotDto> lotDtos, final LotUpdateReques
 	}
 
 	@Override
-	public void saveLotsWithInitialTransaction(final CropType cropType, final Integer userId, final List<LotItemDto> lotItemDtos) {
+	public List<String> saveLots(final CropType cropType, final Integer userId, final List<LotItemDto> lotItemDtos) {
```

Comment:
I think the previous name was more appropiated because we are saving lots and opening a deposit transaction

Reply:
now the initial transaction is optional